05-Time-Series-Raster.Rmd

```diff
@@ -272,37 +300,38 @@ harMetDaily <- read.csv("AtmosData/HARV/hf001-06-daily-m.csv",
 harMetDaily$date <- as.Date(harMetDaily$date, format = "%Y-%m-%d")
 
 #subset out some of the data - 2010-2013 
-yr.09.11_monAvg <- subset(harMetDaily, date >= as.Date('2011-01-01') & date <=
+yr.09.11_dailyAvg <- subset(harMetDaily, date >= as.Date('2011-01-01') & date <=
 as.Date('2012-01-01'))
 
-#as.Date("2006-02-01 00:00:00")
-#plot Some Air Temperature Data
+#plot Air Temperature Data (airt) by julian day (jd)
   
-myPlot <- ggplot(yr.09.11_monAvg,aes(date, airt)) +
+myPlot <- ggplot(yr.09.11_dailyAvg,aes(jd, airt)) +
            geom_point() +
-           ggtitle("Daily Air Temperature\nHarvard Forest") +
+           ggtitle("2011 Air Temperature\nHarvard Forest") +
            theme(plot.title = element_text(lineheight=.8, face="bold",size = 20)) +
            theme(text = element_text(size=20)) +
-           xlab("Time") + ylab("Mean Air Temperature")
+           xlab("Julian Day") + ylab("Mean Air Temperature")
 
-#format x axis with dates
-myPlot + scale_x_date(labels = date_format("%m/%d/%y"))
+myPlot
 
 ```
 
-The temperature data suggests that the pattern of warming and cooling is fairly
-consistent. Maybe we should look at the 3-band RGB imagery to see what that looks like, next. The RGB imagery are located in the `RGB` directory. 
-
-#CHALLENGE
+There are no significant peaks or dips in the temperature during the late summer
+or early fall time period that could logically cause cause the vegetation to
+brown (around day 277) and then quickly green again (by day 309). 
 
-> Open up the RGB images from Julian dates 277 and 293. What do you see?
+What is happening here? 
```

Comment:
Kept the data part of it as a challenge and added a discussion section as follows: 
##Explore the Data Source
The third challenge questions, "Does viewing the RGB imagery from these two days
help explain the NDVI data?" highlights the importance of exploring the source
of any data we work with.  

The NDVI values from day 277 to 293 were very low
suggesting that the vegetation was dead.  However, when we look at the RGB
images from which the NDVI data was derived we see that nearly the entire 
image is filled with clouds.  The very low NDVI values resulted from there being
no growing vegetation in the field of view.